content_sync/pipelines/concourse.py

```diff
@@ -249,3 +259,98 @@ def get_build_status(self, build_id: int):
     def abort_build(self, build_id: int):
         """Abort a build"""
         return self.api.abort_build(build_id)
+
+
+class ThemeAssetsPipeline(BaseThemeAssetsPipeline):
+    """
+    Concourse-CI pipeline for publishing theme assets
+    """
+
+    MANDATORY_SETTINGS = [
+        "CONCOURSE_URL",
+        "CONCOURSE_USERNAME",
+        "CONCOURSE_PASSWORD",
+        "GITHUB_WEBHOOK_BRANCH",
+        "SEARCH_API_URL",
```

Comment:
Also "THEME_ASSETS_PIPELINE" (currently, though that may change if one of the prior suggestions is implemented).